packages/eslint-config/index.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
   },
   rules: {
     curly: ['error', 'all'],
+    'import/no-cycle': 0,
```

Comment:
In the end we didn't want to do any pre-push check for this? Anyways we can think about it and do later

Reply:
I think it could be worth having this enabled, so that we get this lint rule during development.
But instead disabling it during CI (might be possible by checking if env `CI=true`)

Reply:
I am not a fan of spliting and different lint for dev/ci since whatever you don't do in CI is subject to local dev env which leads to inconsistencies. Per this issue here https://github.com/import-js/eslint-plugin-import/issues/2348 they are recommending `{ ignoreExternal: true }`, since we really want to check only our own code. Worth a try if that is a culprit. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md#ignoreexternal

Reply:
Also might be worth a try to play with `maxDepth`, since I think it is Infinity by default.

Reply:
Yes I agree, that's what I wanted to try today before merging

Reply:
So adding this settings adds something like 1 minute to the linting process, do we still want it? tagging also @rebelchris

Reply:
With large instance I managed to keep the same timing. So we'll keep this new setting

Reply:
Let's see if that one can we large and the others maybe less?
(that should even out cost I guess)

Reply:
already fixed the other ones 💪